python/ray/autoscaler/v2/scheduler.py

```diff
@@ -1078,10 +1174,90 @@ class ScheduleResult:
         # Remove the best node from the nodes.
         nodes.pop(best_result.idx)
         logger.debug(
-            "best_node: {}, score: {}, remaining requests: {}".format(
+            "Best node: {}, score: {}, remaining requests: {}".format(
                 best_result.node,
                 best_result.score,
                 ResourceRequestUtil.to_dict_list(best_result.infeasible_requests),
             )
         )
         return best_result.node, best_result.infeasible_requests, nodes
+
+    def _terminate_outdated_nodes(self) -> None:
```

Comment:
i was surprised we have this behavior...

Reply:
it's likely not frequently used.